pkg/profiles/openapi.go

```diff
@@ -104,11 +106,16 @@ func swaggerToServiceProfile(swagger spec.Swagger, namespace, name, clusterDomai
 	return profile
 }
 
-func mkRouteSpec(path, pathRegex string, method string, responses *spec.Responses) *sp.RouteSpec {
+func mkRouteSpec(path, pathRegex string, method string, responses *spec.Responses, exception *spec.Operation) *sp.RouteSpec {
```

Comment:
I think we can clean this up a bit by just having it accept a `*spec.Operation` instead of both a `*spec.Responses` and a `*spec.Operation`.  We can get the responses from the `Operation` that is passed in.  

Why do you name the `Operation` parameter `exception`?  Should this be called something like `operation` or `op` instead?